src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/AddressLogEntry.orm.xml

```diff
@@ -22,7 +22,7 @@
         <field name="objectId" column="object_id" length="64" nullable="true" />
         <field name="objectClass" column="object_class" type="string" />
         <field name="version" column="version" type="integer" />
-        <field name="data" column="data" type="array" />
```

Comment:
Why nullable 🤔?

Reply:
I guess it was done to match original mapping

https://github.com/doctrine-extensions/DoctrineExtensions/blob/8d658b4d22977e3b72f02bfe4e68b2df0ba586aa/src/Loggable/Entity/MappedSuperclass/AbstractLogEntry.php#L83-L89

but I don't think it makes sense to do so at this stage.
```suggestion
        <field name="data" column="data" type="json" />
```

Reply:
I need to leave this change, as without it the removal of address'es is not working
![Screenshot 2024-03-19 at 08 43 13](https://github.com/Sylius/Sylius/assets/53942444/597b7447-b8a4-4f60-8fa1-e2bf8a3b6264)

Reply:
[this scenario](https://github.com/Sylius/Sylius/blob/1.13/features/account/customer_account/address_book/deleting_address.feature) for ref

Reply:
Strange, that it worked before.

What do you think mentioning this change in changelog, if it's necessary?